cpp/examples/downstream/cmake/get_kvikio.cmake

```diff
@@ -30,4 +30,4 @@ function(find_and_configure_kvikio MIN_VERSION)
 
 endfunction()
 
-find_and_configure_kvikio("22.10")
+find_and_configure_kvikio("24.06")
```

Comment:
:rofl:

Reply:
Please update ci/release/update-version.sh so this is kept up to date.